src/Console/Command/DescribeCommand.php

```diff
@@ -253,7 +255,7 @@ private function describeRule(OutputInterface $output, string $name): void
 
         if (0 === \count($codeSamples)) {
             $output->writeln([
-                'Fixing examples cannot be demonstrated on the current PHP version.',
+                'Fixing examples <error>cannot be</error> demonstrated on the current PHP version.',
```

Comment:
This part is not covered with test.

Reply:
indeed, this is status quo of master branch

Reply:
But can be improved... 😉 Not a blocker, though.